tools/nightly.py

```diff
@@ -270,8 +271,8 @@ def activate_command(self) -> str:
         """Get the command to activate the virtual environment."""
         if WINDOWS:
             # Assume PowerShell
-            return f"& {self.prefix / 'Scripts' / 'Activate.ps1'}"
-        return f"source {self.prefix}/bin/activate"
+            return f'''& "{self.prefix / "Scripts" / "Activate.ps1"}"'''
+        return f'source "{self.prefix}/bin/activate"'
```

Comment:
This is unrelated to LD_LIBRARY_PATH update is it? If so, please move to separate PR